### PR TITLE
Create RVRS.Load_VIP_DeathFuneralHomePr.sql

### PR DESCRIPTION
**Pull request by sailendrasingh1**
Funeral Home Code for Review 06/07/2023

### CODE REVIEW

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+*/
+
+	
+PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)	
+					
+   
+		SELECT *
+			
+	        ,CASE WHEN (FNRL_SERVICE_OOS = 'Y' and FH_RESPONSIBLE_NAME= FNRL_NME) THEN 'FNRL_SERVICE_OOS,RESPONSIBE_NAME,FNRL_NME|Error:Funeral Home and Responsible Funeral Home names are same for Trade Service Call' ELSE '' END AS LoadNote_1
+	        ,CASE WHEN (FNRL_SERVICE_OOS = 'N' and FH_RESPONSIBLE_NAME<> FNRL_NME) THEN 'FNRL_SERVICE_OOS,RESPONSIBE_NAME,FNRL_NME|Error:Funeral Home and Responsible Funeral Home names are different for no Trade Service Call' ELSE '' END AS LoadNote_2
```
[the remark this answers is not in the document]
Comment (almabekwoldesenbet) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql:249]: Modify the error msg .
Funeral Home and Responsible Funeral Home names are different for **_none_** Trade Service Call

**Review by almabekwoldesenbet — CHANGES_REQUESTED** [review]
it looks great just a couple of corrections.

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+----------------------------------------------------------------------------------------------------------------------------------------------
+*/
+
+	
+PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)	
+					
+   
+		SELECT *
+			
+	        ,CASE WHEN (FNRL_SERVICE_OOS = 'Y' and FH_RESPONSIBLE_NAME= FNRL_NME) THEN 'FNRL_SERVICE_OOS,RESPONSIBE_NAME,FNRL_NME|Error:Funeral Home and Responsible Funeral Home names are same for Trade Service Call' ELSE '' END AS LoadNote_1
```
[the remark this answers is not in the document]
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql:248]: Fixed

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+	        ,CASE WHEN (FNRL_SERVICE_OOS = 'N' and FH_RESPONSIBLE_NAME<> FNRL_NME) THEN 'FNRL_SERVICE_OOS,RESPONSIBE_NAME,FNRL_NME|Error:Funeral Home and Responsible Funeral Home names are different for no Trade Service Call' ELSE '' END AS LoadNote_2
+	        ,CASE WHEN ((TRADE_FH_UNLISTED =  'N' OR FL_FUNERAL_HOME_UNLISTED = 'N') AND FNRL_NME<>FH_NAME)  THEN 'TRADE_FH_UNLISTED,FL_FUNERAL_HOME_UNLISTED,FNRL_NME,VIP_VT_FUNERAL_HOME_CD.FH_NAME|Warning:The Name of the Funeral Home does not match the Name from Code table' ELSE '' END AS LoadNote_3
+
```
[the remark this answers is not in the document]
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql:251]: Fixed

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql [hunk 4]
```diff
@@ -245,9 +245,11 @@ PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)
...
+	        ,CASE WHEN DimFuneralHomeTypeInternalId =1 AND FNRL_SERVICE_OOS = 'N' and COALESCE(FH_RESPONSIBLE_NAME,'')<>COALESCE(FNRL_NME,'') THEN 'DimFuneralHomeTypeInternalId,FNRL_SERVICE_OOS,RESPONSIBE_NAME,FNRL_NME|Error:Funeral Home and Responsible Funeral Home names are different for Non-Trade Service Call' ELSE '' END AS LoadNote_2
+	        ,CASE WHEN DimFuneralHomeTypeInternalId =1 AND (TRADE_FH_UNLISTED =  'N' OR FL_FUNERAL_HOME_UNLISTED = 'N') AND FNRL_NME<>FH_NAME THEN 'DimFuneralHomeTypeInternalId,TRADE_FH_UNLISTED,FL_FUNERAL_HOME_UNLISTED,FNRL_NME,VIP_VT_FUNERAL_HOME_CD.FH_NAME|Warning:The Name of the Funeral Home does not match the Name from Code table' ELSE '' END AS LoadNote_3
+	        ,CASE WHEN DimFuneralHomeTypeInternalId =1 AND FNRL_NME IS NULL THEN 'DimFuneralHomeTypeInternalId,FNRL_NME|Error:Missing Funeral Home Name' ELSE '' END AS LoadNote_4
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathFuneralHomePr.sql:251]: We have to redone this.  It should be : 
   ,CASE WHEN AND ((FNRL_SERVICE_OOS = 'Y'  AND TRADE_FH_UNLISTED =  'N' ) OR (FNRL_SERVICE_OOS = 'N' FL_FUNERAL_HOME_UNLISTED = 'N')) AND FNRL_NME<>FH_NAME

**Review by almabekwoldesenbet — APPROVED** [review]

**Pull request merged**